atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,6 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        # currently we only support v1+v2 schema1
```

Comment:
I'd previously written this: "To handle these cases, pulp_pull should directly request the v2 schema 2 image manifest from Crane. If the v2 schema 2 image manifest is available, the plugin should take no further action."

By "directly request" I meant that it should contact the registry directly using the Docker Registry HTTP API V2, accepting v2 schema 2 image manifests, yes.